### PR TITLE
feat: GC distribution

### PR DESCRIPTION
**Pull request by lumir-mrkva**

### CODE REVIEW

File: scripts/distribute-into-proxies/index.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  assert.equal(
+    anonymousProxies.length,
+    distribution.length,
+    'not all proxies created',
+  )
+  log('proxies created:', anonymousProxies)
+
+  log('funding proxies...')
+  const transfers = anonymousProxies.map((anon) =>
+    api.tx.balances.transfer(anon, 500 * UNIT),
```
Comment (jak-pan) [inline scripts/distribute-into-proxies/index.js:132]: Is this enough after fee bump? We probably also need sudo here if we do it b4 upgrade.

File: scripts/distribute-into-proxies/index.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+let proxyIndex = 2000
+const multisig = 'bXiGFV6H2JpQYqXFtQtRqXNn7zS22Nco349BE7CvUzBuidraU'
+const UNIT = 1000000000000
+
+const vesting = {
+  start: '13517962',
+  period: '1752',
+  per_period: '',
+  period_count: '6000',
```
Comment (jak-pan) [inline scripts/distribute-into-proxies/index.js:20]: Changed so there are no remainders

File: scripts/distribute-into-proxies/index.js (hunk 1 above)
Comment (lumir-mrkva) [reply scripts/distribute-into-proxies/index.js]: its for storage deposit, which shouldn't change, but we'll test

**Comment by lumir-mrkva** [comment]
prettier sucks, stealing my perfect code...

**Pull request merged**